_data/feats.yml

```
@@ -9,3 +69,418 @@ content:
+      to fade and you may only attempt to reroll a Skill Check once more
+      until the next day when you renew your prayer. If you forget to
+      pray to your deity, they will refuse to grant this ability to
+      you for 1d4 days.
+      
+  - name: Distracting Dance
+    prerequisite: >
+      Be proficient in Performance
+    description: |
+      As a bonus action, you can make a Performance Check with a DC of
+      13+ the Targets Wisdom Modifier, create a small commotion to the
+      target within 30' to allow an ally engaged in combat with the target
+      to make a free Attack of Opportunity with advantage. The target of
+      the distraction must be able to see and hear you for this ability
+      to work. This action acts a ranged spell attack.
+      
+  - name: Eldritch Devotion
+    prerequisite: >
+      Have six levels in Warlock
+    description: |
+      Your powers reach further than ever before and you broaden your
+      horizen of abilities. Gain an additional Eldritch Invocation.
+      Prerequisites still limit selection options of Invocations    
+      
+  - name: Expanded Magic
+    prerequisite: >
+      Have at least one level in Bard, Cleric, Druid, Paladin, Ranger, Sorcerer, or Wizard.
+    description: |
+      During your studies or meditations, you discover similarities and
+      patterns between spells. Upon taking this feat, you learn one new
+      spell of spell level equal to your proficiency bonus or lower. You
+      also know a nuymber of bonus spells equal to your proficiency bonus.    
+      
+  - name: Fortunate Find
+    prerequisite: >
+      Be proficient in Perception or Investigation
+    description: |
+      When performing an investigation or perception check on an indiviual
+      container, body, or item within 5 feet, you gain a +3 bonus to the
+      check. If this find yields gold pieces, gain an extra 1d20 per three
+      character levels. This can only be done once per hour.     
+      
+  - name: Gaia's Ambassador
+    prerequisite: >
+      Be proficient in Animal Handling
+    description: |
+      There's just something about you that intrigues animals. When you are
+      the target of a Beast type creature's attack, you can attempt an
+      Animal Handling check with a DC as listed from below. As a Reaction
+      to dissuade it from attacking you. When dissuaded, the creature will
+      no longer target you for its' current attack action or future attacks
+      unless you attack it. If you succeed on a second Animal Handling check
+      on the creature with a standard action within five minutes of the
+      first check with an increased DC of +6 of the previous check, you gain
+      the creature as an ally for the current combat, or if this is performed
+      out of combat it just leaves you in peace after five minutes. Attacking a
+      convinced creature ends this effect and cannot be performed again for
+      one day.   
+      
+      Tiny:       10
```

```suggestion
      Size       | Check DC
      ---------- | --------
      Tiny       | 10
      Small      | 14
      Medium     | 17
      Large      | 22
      Huge       | 26
      Gargantuan | 31
```

It might be worth putting this in a table for better clarity